src/components/Nav.scss

```
@@ -1,7 +1,78 @@
 @import '../styles/common.scss';
 
-div h1 {
-  font-family: 'Gamja Flower', cursive;
-  color: $color-main;
-  font-size: 50px;
+.navBar {
+  display: flex;
+  justify-content: space-between;
+  position: fixed;
+  align-items: center;
+  height: 60px;
+  width: 100%;
+  background-color: white;
+  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.2);
```

```suggestion
.navBar {
  display: flex;
  justify-content: space-between;
  align-items: center;
  position: fixed;
  height: 60px;
  width: 100%;
  background-color: white;
  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.2);
```
- 연관된 속성끼리 같이 배치해주는게 좋을 것 같습니다.